src/catala/literate/html.ml

```diff
@@ -37,6 +37,34 @@ let raise_failed_pygments (command : string) (error_code : int) : 'a =
     (Printf.sprintf "Weaving to HTML failed: pygmentize command \"%s\" returned with error code %d"
        command error_code)
 
+(** Partial application allowing to remove first code lines of [<td class="code">] and
+    [<td class="linenos">] generated HTML. Basically, remove all code block first lines. *)
+let remove_cb_first_lines : string -> string =
+  R.substitute ~rex:(R.regexp "<pre>.*\n") ~subst:(function _ -> "<pre>\n")
+
+(** Partial application allowing to remove last code lines of [<td class="code">] and
+    [<td class="linenos">] generated HTML. Basically, remove all code block last lines. *)
+let remove_cb_last_lines : string -> string =
+  R.substitute ~rex:(R.regexp "<a.*\n*</pre>") ~subst:(function _ -> "</pre>")
+
+(** Partial application allowing to substitute operators by their unicode representation. *)
+let substitute_arithmetics_op : string -> string =
+  (* (EmileRolley) NOTE: [date] should match the new ISO format. + I don't get why this must be in
```

Comment:
You can delete this date regex, I don't know why it's here. Probably a leftover of something I used to do.